dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/DataModelsTranslations.ts

```diff
@@ -1,6 +1,8 @@
 export const sfdrKpisNameMappings = {
```

Comment:
Has someone actually checked if all the mappings in this file are still reflecting the latest data model for SFDR?

Reply:
I don't think that someone took a deep dive. I corrected some of the entries, but as we have a story concerning the sfdr view panel in the backlog in which we want to make it similiar to lksg and p2p, i dont think we should spend too much time here.

Reply:
For example the cited reports should be visible as a download icon next to the kpis, where they are cited. And the i think the view panel right now, doesn't get the descriptions from the datamodel file but rather from this translation file. That would need to change

Reply:
Ah ok all right 👍 .  I didn't know that.
Then this can be done in the story you mentioned.
Closing this